BlueprintUI/Sources/Layout/ConstrainedAspectRatio.swift

```diff
@@ -0,0 +1,77 @@
+/// Constrains the size of the content element to an aspect ratio.
+public struct ConstrainedAspectRatio: Element {
+    /// Represents whether the content element's size should be expanded to fill its parent
+    /// or shrunk to fit it.
+    public enum ContentMode {
```

Comment:
Doesnt matter within Blueprint, but I tend to make these enums equatable when I can.